### DISCUSSION TITLE
misunderstanding of any_changed inputs

### DISCUSSION THREAD

**Discussion by Benbig09** [tj-actions/changed-files#2256] [Q&A] [answered]
how to get all changes including delete/add/modify in a var like `any_changed`?
i dont understand if the `any_changed` cover all the options, or its only for the changes files.
i want to know if anything has changed in my dir, to trigger something.

**Comment by jackton1** [comment, marked as answer]
Hi @Benbig09 here’s an excerpt from the [documentation](https://github.com/tj-actions/changed-files?tab=readme-ov-file#outputs-) 


`any_changed`
```
Returns true when any of
the filenames provided using the
files* or files_ignore* inputs have changed. This
defaults to true when no
patterns are specified. i.e. includes a combination of all added, copied, modified and renamed files (ACMR).
```

Based on your question 

`any_modified` would be the recommended approach.

```

Returns true when any of
the filenames provided using the
files* or files_ignore* inputs have been modified.
This defaults to true when
no patterns are specified. i.e.
includes a combination of all added, copied, modified, renamed, and deleted files (ACMRD).
```

**Reply by Benbig09** [reply]
thanks a lot!
